packages/veritone-react-common/src/components/formComponents/DateTimePicker.js

```diff
@@ -102,22 +113,21 @@ TimeSelector.propTypes = {
   min: instanceOf(Date),
   max: instanceOf(Date),
   value: string.isRequired,
-  onChange: func.isRequired
+  onChange: func.isRequired,
+  readOnly: bool
 };
 
-const TimeZoneField = ({ value }) => {
-  return (
-    value && (
-      <TextField
-        className={styles.dateTimeTZ}
-        value={value}
-        InputProps={{
-          disableUnderline: true
-        }}
-        disabled
-      />
-    )
-  );
+const TimeZoneField = ({ value, ...rest }) => {
+  return value ? (
+    <TextField
+      className={styles.dateTimeTZ}
+      value={value}
+      InputProps={{
+        disableUnderline: true
+      }}
+      disabled
+    />
+  ) : null;
```

Comment:
pointless change, `value &&` is the correct pattern

Reply:
i recall the component actually needs a value returned (null as this object). otherwise errors will be thrown on undefined content.